ios/AttentiveReactNativeSdk.mm

```diff
@@ -0,0 +1,85 @@
+//
+//  AttentiveReactNativeSdk.m
+//  AttentiveReactNativeSdk
+//
+//  Created by Wyatt Davis on 2/13/23.
+//
+
+#import "AttentiveReactNativeSdk.h"
+#import "attentive-sdk-umbrella.h"
+
+@implementation AttentiveReactNativeSdk {
+    ATTNSDK* _sdk;
+}
+
+RCT_EXPORT_MODULE()
+
+RCT_EXPORT_METHOD(initialize:(NSDictionary*)configuration) {
+    _sdk = [[ATTNSDK alloc] initWithDomain:configuration[@"attentiveDomain"] mode:configuration[@"mode"]];
+    [ATTNEventTracker setupWithSdk:_sdk];
+}
+
+RCT_EXPORT_METHOD(triggerCreative) {
+  dispatch_async(dispatch_get_main_queue(), ^{
+    UIWindow *window = [[UIApplication sharedApplication] keyWindow];
+    UIView *topView = window.rootViewController.view;
```

Comment:
does this mean that the user can't pass in a view to use as the parent view?

Reply:
Correct!